subprojects/gradle/signer/src/gradleTest/kotlin/com/avito/plugin/SignServicePluginTest.kt

```diff
@@ -56,7 +55,31 @@ class SignServicePluginTest {
 
         result.assertThat()
             .buildFailed()
-            .outputContains("host")
+            .outputContains("Invalid signer url value: ''")
+    }
+
+    @Test
+    fun `plugin apply - fails - configuration with invalid url`() {
```

Comment:
`plugin apply - fails`, but we schedule a task: `:app:signApkViaServiceRelease --dry-run` 

It doesn't look like a project configuration. Is it correct?

Reply:
it is, dry run will not run any actual tasks, but goes through configuration phase
i need this combination here, because there is no other way to trigger task configuration